src/utils/file_io.py

```diff
@@ -194,12 +194,11 @@ def create_df_kernel_top_stats(
 
 @demarcate
 def create_df_pmc(
-    raw_data_root_dir, nodes, spatial_multiplexing, kernel_verbose, verbose
+    raw_data_root_dir, nodes, spatial_multiplexing, kernel_verbose, verbose, config
```

Comment:
It is debatable passing profile_config all the way down. Not a big deal for short term.